src/main/flight/mixer.c

```diff
@@ -424,9 +424,11 @@ static void updateDynLpfCutoffs(timeUs_t currentTimeUs, float throttle)
 #endif
 
 static void applyMixerAdjustmentLinear(float *motorMix, const bool airmodeEnabled) {
-    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? motorMixRange : 1.0f;
+    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? 1.0 * motorMixRange : 1.0f;
```

Comment:
Should this be `1.0f / motorMixRange : 1.0f;` as per `applyMixerAdjustmentLinear()`

Reply:
good catch